packages/provider-utils/src/is-abort-error.ts

```diff
@@ -1,6 +1,8 @@
 export function isAbortError(error: unknown): error is Error {
   return (
     error instanceof Error &&
-    (error.name === 'AbortError' || error.name === 'TimeoutError')
+    (error.name === 'AbortError' ||
+      error.name === 'TimeoutError' ||
+      error.name === 'ResponseAborted')
```

Comment:
wonder if we should document which name is triggered when / how

Reply:
is this needed if consume streams swallows all errors?

Reply:
good call!